LSP.sublime-settings

```diff
@@ -47,6 +47,12 @@
   // under the cursor in status bar if available.
   "show_diagnostics_in_view_status": true,
 
+  // Show the diagnostics as inline annotations.
+  // When using the "at-cursor" value it's recommended to not use the "annotation"
+  // value for the "show_code_actions" option as then the code action annotations will
+  // show over the diagnostic annotations.
+  "show_diagnostics_inline": "none",
```

Comment:
Not happy about introducing this option.

I'm thinking that maybe there should be an option like `diagnostics_style` that takes an array and the user can specify multiple values like `inline-after`, `inline-below` or similar.

Maybe there wouldn't be a need for an array if we keep the normal diagnostic highlights (`diagnostics_highlight_style`) out of it.

Though should also think of how to incorporate a setting for showing either all diagnostics at once or only ones intersecting the cursor... Maybe mash it all together into `diagnostics_style` but coming up with nice value names would be a challenge. Something like `inline-after-at-cursor`, `inline-after-all`, `inline-below-at-cursor`, `inline-below-all` sounds absolutely horrendous.